### PR TITLE
Allow SSL in database url

### PR DESCRIPTION
**Pull request by abhay**
Allows adding the "ssl=true" query parameter to DATABASE_URL to try to load SSL. Could be expanded to allow for ssl=required if needed. 

cc: @madninja

### CODE REVIEW

File: src/psql_migration.erl [hunk 1]
```diff
@@ -176,20 +176,33 @@ connection_opts(_Args, {url, DatabaseUrl}) ->
...
+                database => string:slice(Database, 1)},
+
+            case Query of
+                [] -> {ok, ConnectionOpts};
+                "?" ++ QueryString ->
+                    case uri_string:dissect_query(QueryString) of
+                        [] -> {ok, ConnectionOpts};
+                        QueryList ->
+                            case proplists:get_value("ssl", QueryList) of
+                                "true" -> {ok, maps:put(ssl, true, ConnectionOpts)};
```
Comment (madninja) [inline src/psql_migration.erl:201]: why not just merge a map made from the proplist in with ConnectionOpts?

Comment (abhay) [reply src/psql_migration.erl:201]: Would be fine if I trusted developers like me to not put in other garbage in the url string. I’d rather just extract what’s expected.

Comment (madninja) [reply src/psql_migration.erl:201]: Except developers supply the url too? And may have more options than we support out of the box? What are we protecting against?

Comment (abhay) [reply src/psql_migration.erl:201]: Typos mostly. I tend towards defensiveness but it’s a weakly held belief. Will make the change.

Comment (abhay) [reply src/psql_migration.erl:201]: One nice upside of map merging is that we can support ssl=required.

Comment (madninja) [reply src/psql_migration.erl:201]: yeah, I'm with you on only opening the holes one at a time. but that nice upside may make it worth it?

Comment (abhay) [reply src/psql_migration.erl:201]: Other downside of a simple map merge is that we’d have to convert to atoms for ssl and maybe other types for other fields

Comment (madninja) [reply src/psql_migration.erl:201]: ah yeah, fair enough.. let's keep it like this for now

Comment (abhay) [reply src/psql_migration.erl:201]: My madness sometimes has a method. Even if I don’t know it?

Comment (madninja) [reply src/psql_migration.erl:201]: I'd have probably figured out a mad way to get a map done, but it's just not the simplest thing to do here :-)

**Pull request merged**